### PR TITLE
Add a little countdown bar for carousel

### PR DESCRIPTION
**Pull request by AdaRoseCannon**
Add's a small brand colour bar to the carousel to show how long a page has got.

image

### CODE REVIEW

File: client/viewer/js/main.js [hunk 1]
```diff
@@ -149,11 +150,24 @@ window.screensInit = function screensInit() {
...
 
 		if (Carousel.isCarousel(url)) {
 			carousel = new Carousel(url, host);
-			carousel.on('change', updateUrl);
+			carousel.on('change', function (url) {
+				updateUrl(url);
+				DOM.carouselCountdown.style.transition = 'none';
+				DOM.carouselCountdown.style.transform = 'scaleX(1)';
+
+				setTimeout(() => {
```
Comment (seanmtracey) [inline client/viewer/js/main.js:165]: Why in a setTimeout?

File: package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     "express-ftwebservice": "^2.1.2",
     "express-handlebars": "^2.0.1",
     "fetch-er": "0.0.10",
-    "ftlabs-screens-carousel": "github:ftlabs/screens-carousel#v1.0.7",
+    "ftlabs-screens-carousel": "github:ftlabs/screens-carousel#v1.0.9",
```
Comment (seanmtracey) [inline package.json:27]: Is there a reason we don't have the carousel on npm? The viewer module lives there (as well as on GH)...

File: client/viewer/js/main.js (hunk 1 above)
Comment (AdaRoseCannon) [inline client/viewer/js/main.js:165]: To leave time for the css to flush. It'seems bad but there is no way to force this. I tried for ages for a neater solution.

File: package.json (hunk 2 above)
Comment (AdaRoseCannon) [inline package.json:27]: Probably because I didn'the have ftlabs npm push permissions when the module was made.

**Comment by seanmtracey** [comment]
Rightio, LGTM. I'll merge now and deploy on Monday. Will deploy to test in the meantime.

**Pull request merged**